src/main/resources/lang/he_GB.yml

```diff
@@ -0,0 +1,392 @@
+    stage-7:
+      stage-7-title: 'שלד המבנה'
+      stage-7-messages:
+        -  'כעת נוכל סוף סוף להתחיל על המבנה! השלבים הראשונים הם השלד, בהם נתחיל עם הקווים וגובה המבנה.'
+        - 'טלפורט לפחות לֲ§6נקודה אחתֲ§f בקצה הגג כדי ֲ§6לחברֲ§f את הנקודה(ות) לחזית.'
+        - 'השתמש בֲ§6סוגי בלוקים שוניםֲ§f ובצבעים לשלדים כדי להקל על הֲ§6הפרדתֲ§f המבנה לחלקים.'
+        - 'בהמשך נעלה את הקירות ונאטום את הגג. הגיע הזמן לסמן את החלונות והדלתות.'
+        - 'השתמש בפקודת WorldEdit ֲ§6{0}ֲ§f כדי להעלות את הקירות במהירות ובקלות.'
+        - 'מלא את הגג ביד או השתמש בפקודת WorldEdit ֲ§6{1}ֲ§f. לחלופין, השתמש בפקודה ֲ§6{2}ֲ§f כדי להחליף את הבחירה לגגות גדולים ומורכבים יותר.'
+        - 'תמיד בדוק את גובה החלונות והדלתות כך שיתאים לחזית.'
+      stage-7-tasks:
+        - 'קרא את כל הטיפים על המגרש וסמן אותם כנקראים.'
+    stage-8:
+      stage-8-title: 'חלונות'
+      stage-8-messages:
+        - 'שלד המבנה הושלם! נמשיך עם החלונות והדלתות.'
+        - 'אוי... נראה שחסרים שני חלונות. תוכל לעזור לי למקם אותם? הם דומים לאלו בצד ימין.'
+        - 'אל תשכח ֲ§6להכהותֲ§f את ֲ§6החלונותֲ§f והֲ§6דלתותֲ§f כדי שלא תוכל לראות דרכם. אנחנו לא בונים פנים!'
+        - 'יש דרכים רבות לבניית חלונות ל-BTE, למשל באמצעות ֲ§6דגליםֲ§f, ֲ§6סורגיםֲ§f או ֲ§6שטיחיםֲ§f.'
+        - 'השתמש באותם בלוקים כמו החלונות בצד ימין. נסה שוב!'
+        - 'תודה על עזרתך! עכשיו נוכל להמשיך בטקסטורות.'
+      stage-8-tasks:
+        - 'מקם את פרטי החלון החסרים.'
+    stage-9:
+      stage-9-title: 'טקסטורה'
+      stage-9-messages:
+        - 'הטקסטור הוא חלק בלתי נפרד מתהליך הבנייה. חשוב להשתמש בבלוקים ובצבעים נכונים כדי שהמבנה ייראה ריאליסטי.'
+        - 'השתמש בֲ§6Google Street Viewֲ§f או בֲ§6תמונותֲ§f כדי לבחור את הבלוקים הנכונים, מאחר שתמונות מהאוויר לעיתים לא מדויקות.'
+        - 'השתמש בפקודת WorldEdit ֲ§6{0}ֲ§f כדי להחליף בקלות את השלד בתבנית שלך.'
+        - 'נסה להשתמש בֲ§6תערובות בלוקיםֲ§f וֲ§6מעברי צבעֲ§f לקירות ולגג כדי שהמבנה יראה יותר ריאליסטי ובולט.'
+      stage-9-tasks:
+        - 'קרא את כל הטיפים על המגרש וסמן אותם כנקראים.'
+    stage-10:
+      stage-10-title: 'פירוט ושלבים נוספים'
+      stage-10-messages:
+        - 'הפירוט הוא אחד התהליכים החשובים ביותר כי הוא עושה את המבנה ייחודי ומיוחד.'
+        - 'הוסף ֲ§6באנרים מותאמיםֲ§f ו־ֲ§6ראשים מותאמיםֲ§f לבנייה שלך. השתמש בפקודה ֲ§6{0}ֲ§f כדי לקבל מגוון של ראשים מיוחדים.'
+        - 'יש הרבה דרכים ֲ§6לקשטֲ§f את הבניינים שלך. תמיד שים לב לפרטים בחזיתות ובגגות כמו ֲ§6ארובותֲ§f, ֲ§6חלונותֲ§f ו־ֲ§6מרזביםֲ§f.'
+        - 'תודה על ההשתתפות! עכשיו אתה מוכן לבנות בעצמך בפרויקט BuildTheEarth!'
+        - 'לחץ כאן כדי ללמוד עוד על הפרויקט.'
+        - 'כדי <gold>להגיש מועמדות כבונה</gold>, צור והגש <gold>מגרש אחד או יותר</gold> בשרת שלנו. תוכל למצוא מידע נוסף על תהליך ההגשה באתר שלנו או {1}.'
+        - 'אם אתה רוצה לחקור את ההתקדמות של המפה, בקר בשרת <gold>Terra</gold>!'
+        - 'בנייה מהנה! ג˜÷'
+      stage-10-tasks:
+        - 'קרא את כל הטיפים במגרש וסמן אותם כנקראו.'
+# NOTE: Do not change
+config-version: 2.2
```

Comment:
This should be the same number as the one you specified in LangUtil.java.
I would suggest changing this to 1.0 here